### PR TITLE
Reformatted debug mem leak doc

### PR DESCRIPTION
**Pull request by RobinBateman808**
### Motivation:

Documentation isn't formatting correctly, potentially confusing user instructions and examples.

### Modifications:

Corrected syntax and adjusted spacing. 

### Result:

Correct formatting and flow of information.

### CODE REVIEW

**Comment by RobinBateman808** [comment]
@0xTim - Here's the corrected PR. Apologies for the other and causing any confusion. No content changes were made, only syntax to correct the flow of information on the page.

File: documentation/server/guides/memory-leaks-and-usage.md [hunk 1]
```diff
@@ -66,7 +67,9 @@ func myFunctionDoingTheAllocation() {
 }
 
 myFunctionDoingTheAllocation()
+
```
Comment (0xTim) [inline documentation/server/guides/memory-leaks-and-usage.md:70]: Are these linebreaks at the bottom of the codeblocks required (the ones inside as opposed to outside) - doesn't that just add an extra line to all code blocks?

Comment (RobinBateman808) [reply documentation/server/guides/memory-leaks-and-usage.md:70]: Yes. I reverted them back into the syntax and removed characters that I suspect threw the formatting off.

Comment (0xTim) [reply documentation/server/guides/memory-leaks-and-usage.md]: @RobinBateman808 I removed the extra line breaks at the bottom of the code blocks as they were adding extra whitespace and weren't needed. You should be able to run the website locally following the README instructions - just checked and it's rendering fine now

**Review by 0xTim — APPROVED** [review]
Thanks!

**Comment by 0xTim** [comment]
@swift-ci please test

**Pull request merged**